src/tools/haxelib/Data.hx

```diff
@@ -38,6 +38,7 @@ typedef UserInfos = {
 typedef VersionInfos = {
 	var date : String;
 	var name : String;
+	var downloads : Int;
```

Comment:
Adding additional fields that are sent from server to client should not affect the existing haxelib client